dev/conbench_envs/benchmarks.env

```diff
@@ -27,6 +28,7 @@ ARROW_GANDIVA=OFF
 ARROW_HDFS=ON
 ARROW_HOME=$CONDA_PREFIX
 ARROW_INSTALL_NAME_RPATH=ON
+ARROW_JEMALLOC=OFF
```

Comment:
This used to be set [here](https://github.com/voltrondata-labs/arrow-benchmarks-ci/commit/7c20a64e464705d5bf4e2f20f4c8befd601d566a) but I think it belongs in this file instead.